docker/app.dockerfile

```diff
@@ -1,24 +0,0 @@
-FROM node:21.7.3-slim
```

Comment:
you can also consider using a simple HTTP server to serve the artifact, right? 
something like `busybox` (https://lipanski.com/posts/smallest-docker-image-static-website)

Reply:
we could move them to `graphql-api` and serve from there, but I did not want to do it in this PR